src/admin/components/elements/StepNav/index.tsx

```diff
@@ -31,12 +32,14 @@ const StepNav: React.FC = () => {
   const { t, i18n } = useTranslation();
   const dashboardLabel = <span>{t('general:dashboard')}</span>;
   const { stepNav } = useStepNav();
+  const config = useConfig();
+  const { routes: { admin } } = config;
 
   return (
     <nav className="step-nav">
       {stepNav.length > 0
         ? (
-          <Link to="/admin">
+          <Link to={`${admin}`}>
```

Comment:
Only thing I see here is that `admin` doesn't need to be in a template literal and can be just passed as a variable here.

`<Link to={admin}>`

Reply:
a! My bad, will correct that in now.